modules/metastore/metastore.module

```diff
@@ -60,22 +60,23 @@ function resource_mapper_new_revision() {
 }
 
 /**
- * Create a LifeCycle object.
- *
- * @param Drupal\Core\Entity\ContentEntityInterface $entity
+ * Send some entities into their life cycle.
  *
- * @return Drupal\metastore\LifeCycle\Data
- *   LifeCycle object.
+ * @param \Drupal\Core\Entity\EntityInterface[] $entities
+ *   The entities in question.
+ * @param string $stage
+ *   The entity hook stage, such as 'load' or 'update'.
  */
-function metastore_data_lifecycle(EntityInterface $entity, string $stage) {
-
-  if (metastore_entity_is_valid_item($entity)) {
-    $itemFactory = \Drupal::service('dkan.metastore.metastore_item_factory');
-    $metastoreItem = $itemFactory->wrap($entity);
-    \Drupal::service('dkan.metastore.lifecycle')->go($stage, $metastoreItem);
+function metastore_data_lifecycle(array $entities, string $stage): void {
+  /** @var \Drupal\metastore\LifeCycle\LifeCycle $life_cycle */
+  $life_cycle = \Drupal::service('dkan.metastore.lifecycle');
```

Comment:
"lifecycle" is one word. Should not have been camelcase in the first place. But `life_cycle` really looks wrong